src/app/content/hooks/locationChange/buyPrintConfig.ts

```diff
@@ -13,20 +12,9 @@ const loadBuyPrintConfig = (services: MiddlewareAPI & AppServices) => async() =>
     return;
   }
 
-  const response = await buyPrintConfigLoader.load(book)
-    .catch(() => null);
-
-  if (response === null) {
-    Sentry.captureException(new Error(`book: ${book.slug} had no response from buy-book config service`));
-    return;
-  }
-
+  const response = await buyPrintConfigLoader.load(book);
   const config = response.buy_urls[0];
 
-  if (!config) {
```

Comment:
there is a case where we get a response with no buy_urls, this is the case where we want to hide the buy button